runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -53,7 +53,12 @@ template <typename LoaderDomain, typename SpecificLoader> class BaseLoader
   using Tensor = typename LoaderDomain::Tensor;
   using TensorType = typename LoaderDomain::TensorType;
 
+protected:
   bool isOptionalInputTensor(std::int32_t idx) { return idx == -1; }
+  std::vector<BuiltinOperator> getOptionalInputOplist()
+  {
+    return {BuiltinOperator::BuiltinOperator_FULLY_CONNECTED};
```

Comment:
This is tflite's OptionalInput-supporting operation list

Reply:
@glistening 
Following to our offline conversation,
- I've seperated this into tflite_loader's
- So now there is 
  - virtual function for base_loader
  - two sets of overriding function for tflite_loader and circle_loader